/Users/frondeus/Code/project-s/docs/journal/2025-06-21.md:96
```
(let (:a :b :c) (list 1 2 3))
a
```

--- /Users/frondeus/Code/project-s/docs/journal/2025-06-21.md
+++ graphviz-002
@@ -114,7 +114,7 @@
 N7 [label="7 - lvl1 - Variable { id: VarId(0) }\n`(list 1 2 3)`"];
 N8 [label="8 - lvl0 - Tuple { items: [InferedTypeId(1), InferedTypeId(3), InferedTypeId(5)], rest: None }"];
 N9 [label="9 - lvl1 - Applicative { arg: InferedTypeId(8), ret: InferedTypeId(7), first_arg: Some(InferedTypeId(1)) }"];
-N10 [label="10 - lvl1 - Variable { id: VarId(1) }\n`a`"];
+N10 [label="10 - lvl1 - Variable { id: VarId(1) }\n`(top-level (let (:a :b :c) (list 1 2 3)) a)`"];
 N11 [label="11 - lvl1 - Variable { id: VarId(2) }\n`:b`"];
 N12 [label="12 - lvl1 - Variable { id: VarId(3) }\n`:c`"];
 N13 [label="13 - lvl1 - Tuple { items: [InferedTypeId(10), InferedTypeId(11), InferedTypeId(12)], rest: None }\n`(:a :b :c)`"];
