/Users/frondeus/Code/project-s/docs/journal/2025-06-11.md:395
```
(+ (splice (1 2 3)))
```

--- /Users/frondeus/Code/project-s/docs/journal/2025-06-11.md
+++ json-010
@@ -397,7 +397,25 @@
 ```
 
 ```json
-6.0
+Error: Undefined variable: splice
+   ╭─[ <input>:1:5 ]
+   │
+ 1 │ (+ (splice (1 2 3)))
+   │     ───┬──  
+   │        ╰──── Undefined variable: splice
+───╯
+Error: Incompatible types
+   ╭─[ <input>:1:12 ]
+   │
+ 1 │ (+ (splice (1 2 3)))
+   │            ─┬─┬───  
+   │             ╰─────── But got number
+   │               │     
+   │               ╰───── Incompatible types
+   │               │     
+   │               ╰───── Expected function
+───╯
+
 ```
 
 Great!
